### DISCUSSION TITLE
STLs for inner cyclotron diffuser and post

### DISCUSSION THREAD

**Discussion by Shoop64** [gpstar81/GPStar-proton-pack#89] [General]
I noticed after printing that the inner post seems to be too tall and the diffuser doesn't cover entirely.  I made some versions for myself that will hopefully help others.  

Feel free to add these to the repo if you think they'd be useful.

[inner cyclotron alternate stls.zip](https://github.com/gpstar81/haslab-proton-pack/files/11215106/inner.cyclotron.alternate.stls.zip)

**Comment by gpstar81** [comment]
Awesome thanks. Updated.
